kolibri/utils/options.py

```diff
@@ -827,7 +827,6 @@ def _set_from_envvars(conf):
                                 optname=optname, section=section
                             )
                         )
-                    conf[section][optname] = os.environ[envvar]
```

Comment:
This change breaks the functionality of this function - which is called `_set_from_envvars` - we only want to side step this in the case that we are writing to the options.ini file. We should be using the `using_env_vars` dict returned from this function to help us know which values to ignore.

Reply:
Okay i didnt noticed that